kivy/uix/listview.py

```diff
@@ -899,12 +899,18 @@ def __init__(self, **kwargs):
                   item_strings=self.item_strings_changed,
                   adapter=self._trigger_populate)
 
+        self._trigger_bind_adapter = Clock.create_trigger(
+            lambda dt: self.adapter.bind_triggers_to_view(
```

Comment:
you get a new function signature every time with lambda so it is essentially as good as not using a trigger.